service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FeedbackReceiver.java

```diff
@@ -136,7 +137,7 @@ public FeedbackReceiver(
 
     FeedbackReceiver(
             String hostName,
-            TokenCredential credential,
```

Comment:
This change from TokenCredential to TokenCredentialCache is seen a lot in this PR. The reason for this change is that the TokenCredentialCache encapsulates the TokenCredential, but also contains the authentication scope information

Reply:
internal-only impact?

Reply:
Yep